src/backend/main.ts

```diff
@@ -163,9 +164,14 @@ async function initializeWindow(): Promise<BrowserWindow> {
     mainWindow.setFullScreen(true)
   }
 
-  setTimeout(() => {
-    DXVK.getLatest()
-    Winetricks.download()
+  setTimeout(async () => {
+    // Will download Wine if none was found
+    const availableWine = await GlobalConfig.get().getAlternativeWine()
+    Promise.all([
```

Comment:
There might be an `await` missing here

Reply:
to be honest, I don't want to block the whole thread with the download of the tools so doing like this I believe it won't await until everything is finished. Because this is done just after the app was started.

Reply:
If you don't want to wait for the promises to be resolved, why have a `Promise.all` at all then?

Reply:
I don't think this makes any difference when calling them but I wanted to keep all of 3 under the same function just because it looks more organized. In practice it's the same as adding them in line.

Reply:
it's inside a timeout so it shouldn't block the thread anyway

but I agree if we don't care about waiting I don't think we need the Promise.all, it makes it look like something is missing

it does add a bit of extra overhead keeping track of the promises

I understand you want to communicate that the 3 calls are async and they run in parallel so maybe a comment saying something like `// download tools in parallel` instead of wrapping them in Promise.all gives you the same meaning?

Reply:
fixes